measure_extinction/stardata.py

```diff
@@ -1000,6 +1000,64 @@ def _parse_dfile_line(line):
                     colpos = len(line)
                 return (line[0 : eqpos - 1].strip(), line[eqpos + 1 : colpos].strip())
 
+    def get_data(self, req_datasources):
```

Comment:
I can see this is a useful function.  But it could use a more descriptive name so that it is clearer what it is doing.  It is really stripping the identifying info for each dataset and concatenating it into flat arrays.  Maybe something like `get_flat_data`?  Not great, but at least it states it is not just getting the data - which can be done by just using the object structures already.

Reply:
This function is analogous to the `get_fitdata` function in extdata.py, but as the spectrum does not really get fitted, I left out the "fit" part. But yeah, I agree a less vague name could be useful. I will change this.